its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -300,6 +308,62 @@ void shouldRaiseIssuesOnAGoProject() throws IOException {
       assertThat(issueListener.getIssues()).hasSize(1);
     }
 
+    @Test
+    void shouldRaiseIssuesOnACloudFormationProject() throws IOException {
```

Comment:
As most of the findings raised by IaC are security hotspots, you need to call `enableHotspots()` on the engine